src/services/autoLoginService.js

```diff
@@ -39,8 +39,22 @@ class AutoLoginService {
         store.dispatch(upsertUser(userData));
 
         currentPath &&
-          store.dispatch(setSelectedConversation({ id: currentPath.slice(1) }));
+          store.dispatch(
+            setSelectedConversation({ id: currentPath.split("/")[0].slice(1) })
+          );
         store.dispatch(setUserIsLoggedIn(true));
+
+        const token = localStorage.getItem("sessionId");
+        if (token && token !== "undefined") {
```

Comment:
do we really need it?

we already set it `      localStorage.setItem("sessionId", userToken);` above, 
so it always will be non-empty